### DISCUSSION TITLE
Ability to set screen margins

### DISCUSSION THREAD

**Discussion by neutonfoo** [rxhanson/Rectangle#1004] [Ideas]
Recently started using https://github.com/FelixKratz/SketchyBar. It would be amazing to allow users to set margins on the top/right/bottom/left of the screen so expanded and resized windows don't cover a section of the screen (so the bar will still be visible).

**Comment by Aaron-Rumpler** [comment]
This option already exists, but isn't exposed in the GUI. See instructions [here](https://github.com/rxhanson/Rectangle/blob/master/TerminalCommands.md#setting-gaps-at-the-screen-edges).

**Reply by neutonfoo** [reply]
Oh interesting, didn't know about this page. Thanks!
